Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2010-09-03
Initial Package Version: 0.25
Upstream Status: Applied
Origin: Ralf Wildenhues, Adam Mercer
Description: Fix quoting so that autoconf-2.66+ does not die when
running autogen.sh or autoreconf.

diff -Naur a/pkg.m4 b/pkg.m4
--- a/pkg.m4	2009-12-06 20:39:02.000000000 +0000
+++ b/pkg.m4	2010-08-19 07:54:09.000000000 +0100
@@ -135,7 +135,7 @@
 Consider adjusting the PKG_CONFIG_PATH environment variable if you
 installed software in a non-standard prefix.
 
-_PKG_TEXT])dnl
+_PKG_TEXT])[]dnl
         ])
 elif test $pkg_failed = untried; then
      	AC_MSG_RESULT([no])
@@ -146,7 +146,7 @@
 
 _PKG_TEXT
 
-To get pkg-config, see <http://pkg-config.freedesktop.org/>.])dnl
+To get pkg-config, see <http://pkg-config.freedesktop.org/>.])[]dnl
         ])
 else
 	$1[]_CFLAGS=$pkg_cv_[]$1[]_CFLAGS
